src/app/components/ExperimentalEOJ/Promo/index.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useContext } from 'react';
+import { shape, string, func } from 'prop-types';
+import styled from '@emotion/styled';
+import path from 'ramda/src/path';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { getSerifBold } from '@bbc/psammead-styles/font-styles';
+import {
+  C_GREY_2,
+  C_GREY_6,
+  C_GREY_8,
+  C_WHITE,
+} from '@bbc/psammead-styles/colours';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_0_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { ServiceContext } from '#contexts/ServiceContext';
+import useOperaMiniDetection from '#hooks/useOperaMiniDetection';
+
+const Link = styled.a`
+  ${({ script }) => script && getPica(script)}
+  ${({ service }) => service && getSerifBold(service)}
+  width: 100%;
+  overflow-wrap: break-word;
+  text-overflow: ellipsis;
+  text-decoration: none;
+
+  overflow-x: hidden;
+  overflow-y: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+
+  color: ${C_GREY_8};
+  &:visited {
+    color: ${C_GREY_6};
+  }
+`;
+
+const PromoBox = styled.div`
+  background-color: ${C_GREY_2};
+  padding: ${GEL_SPACING_DBL};
+  margin-bottom: ${GEL_SPACING_TRPL};
+  @media (min-width: ${GEL_GROUP_0_SCREEN_WIDTH_MIN}) {
+    width: 14.8rem;
+  }
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    width: 11.1rem;
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    width: 12.6rem;
+  }
+`;
+
+const OperaPromoBox = styled.div`
+  background-color: ${C_WHITE};
```

Comment:
The Colour should be changed here too. I will update this ;)

Reply:
Ah good spot! Thanks, updated now 👍🏼

Reply:
I should have created a constant called `cardsColor` or something now that I notice 😂... Never too late 👀